### PR TITLE
chore: add T.kumaran

### PR DESCRIPTION
**Pull request by synackred**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1018"></a>

### CODE REVIEW

File: public/data/kumaran.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      {
+        "title": "Started Freelancing,Pentesting,Bugbounty Hunting",
+        "date": "september 2020",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing"
+      }
+ ]
+
+}
```
Comment (krshdev) [inline public/data/kumaran.json:28]:
```suggestion
}

```

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please see comments above by @krshdev

File: public/data/kumaran.json (hunk 1 above)
Comment (synackred) [reply public/data/kumaran.json]: yeah changed

File: public/data/kumaran.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing,Pentesting,Bugbounty Hunting",
```
Comment (Panquesito7) [inline public/data/kumaran.json:20]: I think it'd look better like this 😅 What do you think?
```suggestion
        "title": "Started Freelancing, Pentesting, and Bugbounty Hunting",
```

**Comment by emmalearnscode** [comment]
You currently have 2 pull requests open for your profile. Please close one of them.

**Comment by synackred** [comment]
yeah closed

File: public/data/kumaran.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing, Pentesting,and Bugbounty Hunting",
```
Comment (krshdev) [inline public/data/kumaran.json:20]: Maybe you missed the space here 😅 

```suggestion
        "title": "Started Freelancing, Pentesting, and Bugbounty Hunting",
```

Comment (synackred) [reply public/data/kumaran.json]: yeah space added to it

File: public/data/kumaran.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing, Pentesting, and Bugbounty Hunting",
+        "date": "september 2020",
```
Comment (Panquesito7) [inline public/data/kumaran.json:21]: What do you think if we capitalize the `s` here? 😅
```suggestion
        "date": "September 2020",
```

Comment (synackred) [reply public/data/kumaran.json]: done

**Review by Panquesito7 — APPROVED** [review]
Thanks for your patience and contribution! 🚀

**Comment by synackred** [comment]
Thanks

**Pull request merged**